assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -14,11 +15,13 @@ import { Link, useNavigate, useSearchParams } from "react-router-dom";
 import cx from "classnames";
 import useSWR, { mutate } from "swr";
 import { useRouteToRouteIDsMap } from "Hooks/useRouteToRouteIDsMap";
+import KebabMenu from "Components/KebabMenu";
+import moment from "moment";
 import { updateExistingPaMessage } from "Utils/api";
 import { UpdatePaMessageBody } from "Models/pa_message";
-import Toast from "Components/Toast";
+import Toast, { type ToastProps } from "Components/Toast";
 
-type StateFilter = "current" | "future" | "past";
+type StateFilter = "current" | "future" | "done";
```

Comment:
Because these values are exposed in the URL, this will break any existing open tabs/bookmarks to the "Past/Done" page (as we just recently experienced with renaming `active` to `current`). Can we keep these values the same, and just change how the UI refers to it? I think it's probably fine if the URL still says "past", and `past` as an internal name might actually make more sense for us due to the symmetry with `future`.

Reply:
Good callback. I can change these back.